modules/gdscript/register_types.cpp

```diff
@@ -36,10 +36,17 @@
 #include "gdscript_tokenizer.h"
 #include "gdscript_utility_functions.h"
 
+#ifdef TOOLS_ENABLED
+#include "editor/gdscript_highlighter.h"
+#include "editor/gdscript_translation_parser_plugin.h"
+
+#ifndef GDSCRIPT_NO_LSP
+#include "language_server/gdscript_language_server.h"
+#endif
+#endif
```

Comment:
This case is a bit pathological with nested conditions mirrored for local and core includes :|

A few other `register_types.cpp` are like this, but overall it's not too bad.